understanding/22/consistent-help.html

```diff
@@ -122,7 +124,7 @@ <h2>Techniques for Consistent Help</h2>
       <section id="sufficient">
          <h3>Sufficient Techniques for Consistent Help</h3>
          <ol>
-            <li><a href="../../techniques/general/G220" class="general">Provide a contact-us link</a></li> 
+            <li><a href="../../techniques/general/G220" class="general">Provide a contact-us link in a consistent location</a></li> 
```

Comment:
The link text should contain the "G220:" text to be consistent with other understanding docs.

Reply:
I was simply updating the technique. I leave @michael-n-cooper to decide if the number should be included. I thought that was accomplished by a script.